exasol_integration_test_docker_environment/test/test_test_environment_db_disk_size.py

```diff
@@ -18,57 +20,45 @@ def setUpClass(cls):
                 utils.INTEGRATION_TEST_DOCKER_ENVIRONMENT_DEFAULT_BIN,
                 clean_images_at_close=False)
 
-    def setUp(self):
-        self.client = docker.from_env()
-
     @classmethod
     def tearDownClass(cls):
-        try:
-            cls.test_environment.close()
-        except Exception as e:
-            print(e)
+        close_environments(cls.test_environment)
+
+    def setUp(self) -> None:
+        self.docker_envs = ()
 
     def tearDown(self):
-        try:
-            self.on_host_docker_environment.close()
-        except Exception as e:
-            print(e)
-        try:
-            self.google_cloud_docker_environment.close()
-        except Exception as e:
-            print(e)
-        self.client.close()
+        close_environments(*self.docker_envs)
```

Comment:
I think, it is better to don't store the environments and self and clean them up in the test function, because otherwise we create "class global state" and the cleanup is then not directly obvious and the border between fixture and test blurs

Reply:
Done